### PR TITLE
ON-1379 consistent user ids

### PR DESCRIPTION
**Pull request by martin-hornsby**

### CODE REVIEW

File: src/SFA.DAS.ApplyService.Application/Apply/Review/Return/ReturnRequestHandler.cs [hunk 1]
```diff
@@ -76,14 +76,14 @@ private async Task NotifyContact(Guid applicationId, int sequenceId)
             if (sequenceId == 1)
             {
                 var lastInitSubmission = application.ApplicationData?.InitSubmissions.OrderByDescending(sub => sub.SubmittedAt).FirstOrDefault();
-                var contactToNotify = await _contactRepository.GetContact(lastInitSubmission?.SubmittedBy);
+                var contactToNotify = await _contactRepository.GetContact(lastInitSubmission.SubmittedBy);
```
Comment (MarkFCain) [inline src/SFA.DAS.ApplyService.Application/Apply/Review/Return/ReturnRequestHandler.cs]: Is this a wise update? Code earlier indicates lastInitSubmission may be null

File: src/SFA.DAS.ApplyService.Application/Apply/Review/Return/ReturnRequestHandler.cs [hunk 2]
```diff
@@ -76,14 +76,14 @@ private async Task NotifyContact(Guid applicationId, int sequenceId)
...
 
                 await _emailServiceObject.SendEmailToContact(EmailTemplateName.APPLY_EPAO_UPDATE, contactToNotify, new { reference });
             }
             else if (sequenceId == 2)
             {
                 var lastStandardSubmission = application.ApplicationData?.StandardSubmissions.OrderByDescending(sub => sub.SubmittedAt).FirstOrDefault();
-                var contactToNotify = await _contactRepository.GetContact(lastStandardSubmission?.SubmittedBy);
+                var contactToNotify = await _contactRepository.GetContact(lastStandardSubmission.SubmittedBy);
```
Comment (MarkFCain) [inline src/SFA.DAS.ApplyService.Application/Apply/Review/Return/ReturnRequestHandler.cs]: See previous comment

File: src/SFA.DAS.ApplyService.Application/Apply/Review/Return/ReturnRequestHandler.cs (hunk 1 above)
Comment (martin-hornsby) [reply src/SFA.DAS.ApplyService.Application/Apply/Review/Return/ReturnRequestHandler.cs]: Added a null check

File: src/SFA.DAS.ApplyService.Application/Apply/Review/Return/ReturnRequestHandler.cs (hunk 2 above)
Comment (martin-hornsby) [reply src/SFA.DAS.ApplyService.Application/Apply/Review/Return/ReturnRequestHandler.cs]: Added a null check

**Pull request merged**